.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
.DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file 👍